### PR TITLE
macOS: Update Qt

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Update scripts
- [x] Fix RPATH issues
- [x] Upload package on the FTP and update MD5
- [x] Fix crash on startup
- [x] Fix warning
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Notes for the reviewer:

- I choose Qt 5.12.5 on macOS instead of Qt 5.13.1 because it's not yet available as a precompiled package on macOS. Qt 5.12.5 is the [last LTS](https://en.wikipedia.org/wiki/Qt_version_history) and I don't want to rebuild manually Qt if I can avoid this.
- I dropped the general OpenGL setup in main.cpp. It is causing issue in QWebEngine (mainly, a warning complaining about the use of core profile). The target OpenGL format (OpenGL 3.3, etc.) is set in WbWrenWindow and it seems to be sufficient.

**Comment by fabienrohrer** [comment]
👍 ✔️

**Comment by omichel** [comment]
Just a preliminary question, it seems there is a pre-compiled binary of Qt 5.13.1 for mac here: https://www.qt.io/offline-installers : Is there a problem with it?

**Comment by fabienrohrer** [comment]
Good point, I missed this page. Usually I use the online installer which doesn't include this package:

Capture d’écran 2019-10-30 à 10 31 34

I will check if this is equivalent.

**Comment by fabienrohrer** [comment]
Fortunately, all the issues I solved for Qt 5.12 are present in Qt 5.13 too.
It remains only to wait on the distrib test results: https://travis-ci.com/cyberbotics/webots

File: src/install_scripts/qt_mac_installer.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
-QT_VERSION=5.11.2
-WEBOTS_PATH=~/develop/webots
+WEBOTS_PATH=~/webots
```
Comment (omichel) [inline src/install_scripts/qt_mac_installer.sh]: Wouldn't it be more generic/scalable to use WEBOTS_HOME here?

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**